api/lib/session/bloc/session_bloc.dart

```diff
@@ -21,10 +22,15 @@ class SessionBloc extends BroadcastBloc<SessionEvent, SessionState> {
   }
 
   void _onPlayerAdded(OnPlayerAdded event, Emitter<SessionState> emit) {
+    // Round has started when the first player joins the game
+    if (state.players.isEmpty) {
+      emit(state.copyWith(correctAnswer: getRandomWord));
```

Comment:
I think we should start the round when atleast two players have joined. Or we can have a start button, what say?

Reply:
Also the correct answer is changing when a new player joins

Reply:
Let's make this case **_insensitive_** (uppercase,lowercase)

Reply:
lets consider this, the first player who joined make it admin of the particular sessio. Who can start the session after all the players has joined the game

Reply:
for now, the round autostarts when 2 players have joined,
Will target "Start round" feature in next PR.